src/libraries/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpResponseStream.cs

```diff
@@ -142,12 +146,17 @@ private async Task CopyToAsyncCore(Stream destination, byte[] buffer, Cancellati
                     int bytesRead = await _state.LifecycleAwaitable;
                     if (bytesRead == 0)
                     {
+                        ReadResponseTrailers();
                         break;
                     }
                     Debug.Assert(bytesRead > 0);
 
                     // Write that data out to the output stream
+#if NETSTANDARD2_1
+                    await destination.WriteAsync(buffer.AsMemory(0, bytesRead), cancellationToken).ConfigureAwait(false);
```

Comment:
Are there any observable advantages from using this overload of `Stream.WriteAsync` explicitly? I would expect that the performance of (most) underlying implementations is identical. Or am I missing something?

Reply:
There is an error when not calling it on netstandard2.1:
> error CA1835: Change the 'WriteAsync' method call to use the 'Stream.WriteAsync(ReadOnlyMemory<byte>, CancellationToken)' overload